src/zkp/pisch.rs

```diff
@@ -24,13 +24,13 @@ use utils::CurvePoint;
 
 #[derive(Debug, Serialize, Deserialize)]
 pub(crate) struct PiSchProof {
-    pub(crate) A: CurvePoint,
-    e: BigNumber,
-    z: BigNumber,
+    pub(crate) commitment: CurvePoint,
+    challenge: BigNumber,
+    response: BigNumber,
 }
 #[derive(Debug, Serialize, Deserialize)]
 pub(crate) struct PiSchPrecommit {
-    pub(crate) A: CurvePoint,
+    pub(crate) precommitment: CurvePoint,
     alpha: BigNumber,
```

Comment:
todo: pick a better name for this. `alpha` doesn't mean anything if you're not looking at the paper to see what it is.